src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -334,7 +336,17 @@ public void onCompletion(HttpResponse httpResponse, Object result, Throwable e)
                             // The entire _bulk_get is finished:
                             if (e != null) {
                                 setError(e);
-                                completedChangesCount.addAndGet(remainingRevs.size());
+                                Log.e(TAG, "RemoteRequestCompletionBlock: remainingRevs.size()=" + remainingRevs.size());
+                                //completedChangesCount.addAndGet(remainingRevs.size());
+                                for (int i = 0; i < remainingRevs.size(); i++) {
+                                    RevisionInternal rev = remainingRevs.get(i);
+                                    Log.e(TAG, "RemoteRequestCompletionBlock: rev="+rev);
```

Comment:
Same as above. The log message doesn't refer to the error. Should it be Log.v instead?